Multisig/UI/ClaimToken/ClaimSafeTokenFlow.swift

```diff
@@ -125,7 +146,7 @@ class ClaimSafeTokenFlow: UIFlow {
     }
 
     func chooseDelegate() {
-        delegateFlow = SelectDelegateFlow(safe: safe, guardian: selectedGuardian, customAddress: selectedCustomAddress, factory: factory, completion: { [unowned self] _ in
+        delegateFlow = SelectDelegateFlow(safe: safe, controller: controller, factory: factory, completion: { [unowned self] _ in
```

Comment:
I think we need to have controller as shared instance instead of passing it all over the views

Reply:
🤷 could be, I'd prefer to keep it like this for now, it's injected from outside. It will be initialized with different configuration for rinkeby or mainnet in a later task, so we can reconsider it there.